src/main/java/com/billennium/bcvsaver/entity/Language.java

```diff
@@ -0,0 +1,19 @@
+package com.billennium.bcvsaver.entity;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Data
+public class Language {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    // it will be enum later
```

Comment:
Add `TODO` with comments like this one ;-)
IntelliJ can show you all comments with TODO.
`//TODO: it will be enum later`